explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import Longtext from '../../components/longtext/Longtext';
+import Search from '../../components/search/Search';
+import theme from '../../styles/theme.module.css';
+import { DefaultRpcClient as rpc } from '../../utils/api/DefaultRpcClient';
+import ErrorResult from '../error-result/ErrorResult';
+
+import styles from './RecentTxCard.module.css';
+
+const initState = {
+    loadState: 'pending',
+    lastestTx: [],
+};
+
+const getRecentTransactions = async (txNum: number) => {
+    try {
+        // Get the lastest transactions
+        // TODO add batch transaction kind
+        // TODO sui.js to get the lastest transactions meta data
+        const transactions = await rpc
+            .getRecentTransactions(txNum)
+            .then((res: any) => res);
+        const txLatest = await Promise.all(
+            transactions.map(async (tx: any) => {
+                //
+                const txData = await rpc
+                    .getTransaction(tx[1])
+                    .then((res: any) => res)
+                    .catch((err: any) => false);
+
+                // For tx with errors or not found
+                // return false and skip the transaction
+                if (!txData) {
+                    return false;
+                }
+                const txKind = Object.keys(
+                    txData.transaction.data.kind.Single
+                )[0];
+
+                return {
+                    block: tx[0],
```

Comment:
rename this to sequence_id, if this is used somewhere, otherwise just remove this field.

Reply:
@666lcz are you saying we will never use the `sequence_id` or "blockId" ?